iris/tests/download_manager/downloads_private_window_not_leak_non_private_window.py

```diff
@@ -89,3 +92,8 @@ def run(self):
         click(DownloadManager.DownloadsPanel.DOWNLOADS_BUTTON)
         expected = exists(DownloadManager.DownloadsPanel.NO_DOWNLOADS_FOR_THIS_SESSION, 10)
         assert_true(self, expected, 'There are no downloads displayed in the Downloads Panel.')
+
+    def teardown(self):
+        # Cleanup.
+        path = IrisCore.get_downloads_dir()
+        downloads_cleanup(path)
```

Comment:
download_cleanup() function shouldn't have a path parameter. The name is self-explanatory that it refers to the downloads directory